openassessment/xblock/grade_mixin.py

```diff
@@ -106,9 +110,9 @@ def render_grade_complete(self, workflow):
         if "peer-assessment" in assessment_steps:
             feedback = peer_api.get_assessment_feedback(submission_uuid)
             peer_assessments = [
-                self._assessment_grade_context(asmnt)
-                for asmnt in peer_api.get_assessments(submission_uuid)
-            ]
+                self._assessment_grade_context(peer_assessment)
+                for peer_assessment in peer_api.get_assessments(submission_uuid)
+                ]
```

Comment:
Nice catch! I don't know why this didn't cause any issues. We really need to get PyLint/PEP8 running...